packages/calypso-build/webpack/mini-css-with-rtl.js

```diff
@@ -0,0 +1,45 @@
+const webpack = require( 'webpack' );
```

Comment:
This is equivalent to our patch to `mini-css-extract-plugin` to support RTL (https://github.com/Automattic/mini-css-extract-plugin-with-rtl/compare/87d929ef0bab60cca36710cc74b8eb23b77c7bd2...Automattic:v0.8.0-with-rtl)